dbux-code/src/commands/runCommands.js

```diff
@@ -46,6 +46,9 @@ function getArgs(debugMode) {
   let env = config.get(`dbux.${runMode}.env`);
   dbuxArgs += `${parseEnv(env)}`;
 
+  let packageWhitelists = config.get(`dbux.packageWhitelist`);
+  if (packageWhitelists) dbuxArgs += packageWhitelists.split(' ').map(packageName => ` -w ${packageName}`).join(' ');
```

Comment:
Oh! Right... we cannot actually use space because of a limitation in how we handle arguments in dbux-cli. I only now understood the issue here.

In this case, we can keep it this way, or switch back to comma-separated values, after fixing it in `buildBabelOptions` mentioned above :)

Reply:
The second option sounds great to me. I'll change it back to use this method.